packages/documentsearch/src/searchview.tsx

```diff
@@ -282,34 +287,32 @@ function SearchIndices(props: ISearchIndexProps) {
 }
 
 interface IFilterToggleProps {
-  enabled: boolean;
-  toggleEnabled: () => void;
+  visible: boolean;
+  toggleVisible: () => void;
+  anyEnabled: boolean;
```

Comment:
Function of `anyEnabled` variable was not evident to me from its name. Would using `filterEnabled` or `anyFilterenabled` instead improve clarity?